.github/workflows/buildAndTest.yml

```diff
@@ -213,31 +321,13 @@ jobs:
           fi
           echo "DIFF_COMMIT=$DIFF_COMMIT_SHA" >> $GITHUB_ENV
 
-      # Run 'git clang-format', comparing against the target commit hash. If
-      # clang-format fixed anything, fail and output a patch.
-      - name: clang-format
-        if: ${{ always() }}
-        run: |
-          # Run clang-format
-          git clang-format $DIFF_COMMIT
-          git diff --ignore-submodules > clang-format.patch
-          if [ -s clang-format.patch ]; then
-            echo "Clang-format found formatting problems in the following " \
-              "files. See diff in the clang-format.patch artifact."
-            git diff --ignore-submodules --name-only
-            git checkout .
-            exit 1
-          fi
-          echo "Clang-format found no formatting problems"
-          exit 0
-
       # Run clang-tidy against only the changes. The 'clang-tidy-diff' script
       # does this if supplied with the diff.
       - name: clang-tidy
         if: ${{ always() }}
         run: |
           git diff -U0 $DIFF_COMMIT | \
-            clang-tidy-diff -path build_assert -p1 -fix
+            clang-tidy-diff -path build_clang -p1 -fix
```

Comment:
I neglected to fail this step if clang-tidy fails... You might make that change as well.